packages/plugins/actions/actions-core/src/actions/Throw.js

```diff
@@ -27,6 +27,9 @@ class ThrowActionError extends Error {
 }
 
 function Throw({ methods: { getBlockId, getPageId }, params }) {
+  if (!type.isObject(params)) {
+    throw new Error(`Invalid Throw, check action params. Received "${JSON.stringify(params)}".`);
```

Comment:
If the test throws for params, not an object, can we change the error message to say that params must be an object?

Reply:
Yes we can. I originally refactored so that we only have one throw, but then the test where the `throw` property is a string does not throw like before. I think we should definitly ensure that `throw` is a boolean or none.

I will update this so that those are checked better, with better error messages.